.github/workflows/pr-e2e.yml

```diff
@@ -77,6 +85,16 @@ jobs:
     concurrency: pr-e2e-tests
     if: needs.triage.outputs.run-e2e == '1'
     steps:
+      - uses: actions/github-script@v6
+        name: Remove ok-to-merge label
+        with:
+          script: |
+            github.rest.issues.removeAllLabels({
```

Comment:
Same here

Reply:
I have discovered that gh-cli solves this problem, so I have updated the PR